src/app/app.module.ts

```diff
@@ -44,10 +46,12 @@ const config: OktaConfig = {
   imports: [
     AppRoutingModule,
     BrowserModule,
+    HttpClientModule,
```

Comment:
This is a one time requirement for making the http client module available to other services and components